frontend/app/tests/e2e/pages/history-page/trade-history-page.ts

```diff
@@ -17,6 +17,7 @@ export class TradeHistoryPage {
     return () => {
       // Wait for response.status to be 200
       cy.wait('@apiCall').its('response.statusCode').should('equal', 200);
+      cy.wait(500);
```

Comment:
Any other way to fix it than hardcoding a wait period?

Reply:
I'm not sure, I can't find other easy way for fixing this. The problem is sometimes, the order of flow is like this:
call api -> cypress get some element -> the api changes is refresh whole table -> the previous element that cypress get is gone now.